scripts/generators/assets/docs_only_fields.yml

```diff
@@ -0,0 +1,45 @@
+process:
+  fields:
+    process.entry_meta.type:
```

Comment:
Is there a way that this can be implemented without duplicating the definitions from schemas/process.yml? I think this will be error prone in that someone makes a change to one and forgets to update the other.

Are there use case for this feature for fields that are not part of the schema? Perhaps there could be an attribute in `ecs/schemas/subset/main.yml` that could indicate a field should only be added to docs.

```
process:
    fields:
      same_as_process:
        docs_only: true
```